ludwig/utils/strings_utils.py

```diff
@@ -282,31 +283,49 @@ def __init__(self, **kwargs):
     def __call__(self, text):
         pass
 
-
 class CharactersToListTokenizer(BaseTokenizer):
     def __call__(self, text):
         return text
 
-
 class SpaceStringToListTokenizer(BaseTokenizer):
     def __call__(self, text):
         return SPLIT_REGEX.split(text.strip())
 
+    @staticmethod
```

Comment:
We should handle the case where the `call_inference` method is not implemented. One way to do this is within `BaseTokenizer`:

```
def call_inference(t: tf.Tensor) -> tf.Tensor:
    raise RuntimeError(f'Graph inference not supported for tokenizer: {self.__class__.__name__)')
```